.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+If you're adding a new endpoint consider the following
+ * [ ] Note that the list is sorted alphabetically
+ * [ ] Verify that your endpoint is valid using <https://validator.spaceapi.io/ui>
```

Comment:
I think a lot of people don't know about the validator and this is a good moment to make them aware of it.

Reply:
Generally a good idea, but if we want to do this as a checklist, I'd formulate it as "This is completed", e.g. `The endpoint is valid according to <https://validator.spaceapi.io/ui>`, `The new entry is at the correct (alphabetically sorted) line`.

Reply:
@rnestler is this something we can look it soon? We've had multiple directory PRs since, and in (close to) all of them we had to refer people to the validator.

Reply:
Fixed it. I'd encourage you to use the "Add a suggestion" feature if you want to change small wording things, such that one can just apply the change and then merge it.